### PR TITLE
Added imageFeatures, imageformat and fstype so that it works with snapshots

### PR DESCRIPTION
**Pull request by hoeghh**
**Description of your changes:**
Added imageFeature, imageFormat and fstype to Block Storage example, so that snapshots will work.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [ ] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.
[skip ci]

### CODE REVIEW

**Comment by Madhu-1** [comment]
@hoeghh DCO is failing https://github.com/rook/rook/pull/3670/checks?check_run_id=200458647
can you add sign-off  to the commit

**Comment by Madhu-1** [comment]
added skip ci to the PR

**Comment by Madhu-1** [comment]
@travisn @leseb PTAL

File: Documentation/ceph-block.md [hunk 1]
```diff
@@ -52,11 +52,23 @@ parameters:
     clusterID: rook-ceph
     # Ceph pool into which the RBD image shall be created
     pool: replicapool
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
```
Comment (travisn) [inline Documentation/ceph-block.md]: @Madhu-1 If it defaults to "2", why do we need it here in the doc? The general approach has been to keep documentation examples as simple as possible by not including defaults.

**Pull request merged**